src/RemoveArrayKey.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace Yiisoft\Arrays;
+
+class RemoveArrayKey
```

Comment:
Set value to an instance of of RemoveArrayKey in order for ArrayHelper::merge() to remove such key from resulting array.